data/docs/install/azure-container-apps.mdx

```diff
@@ -0,0 +1,1237 @@
+3. **Azure CLI version** 2.62 or higher
+
+---
+
+## Setup Azure Infrastructure
+
+### 1. Install Container Apps Extension
+
+Install the [Azure Container Apps CLI extension](https://learn.microsoft.com/azure/container-apps/azure-cli):
+
+```bash
+az extension add --name containerapp
+```
+
+For detailed setup instructions, refer to the [getting started guide](https://learn.microsoft.com/azure/container-apps/get-started?tabs=bash#setup).
+
+### 2. Configure Environment Variables
+
+Set up your deployment variables. Replace the placeholder values with your own:
+
+```bash
+# Your configuration
+SUBSCRIPTION_ID=<YOUR_SUBSCRIPTION_ID>
+RG_NAME=<YOUR_RESOURCE_GROUP>
+LOCATION=<YOUR_LOCATION>                    # Choose a supported region
+STORAGE_ACCOUNT=<uniquestorageacct>  # Lowercase letters and numbers only
+ENV_NAME=signoz
+FILE_SHARE_NAME=signoz
+```
+
+> **Note**: The storage account name must be globally unique, contain only lowercase letters and numbers, and be between 3-24 characters. See [supported regions](https://learn.microsoft.com/azure/container-apps/quotas#region-availability) for available locations.
+
+### 3. Create Resource Group and Container Apps Environment
+
+Create the foundational Azure resources:
+
+```bash
+# Set your subscription (if not already configured)
+az account set --subscription $SUBSCRIPTION_ID
+
+# Create resource group
+az group create \
+  --name $RG_NAME \
+  --location $LOCATION
+
+# Create Container Apps environment
+az containerapp env create \
+  --name $ENV_NAME \
+  --resource-group $RG_NAME \
+  --location $LOCATION
+```
+
+### 4. Create Storage Account and File Share
+
+ClickHouse requires persistent storage for configuration files. Learn more about [Azure Files with Container Apps](https://learn.microsoft.com/azure/container-apps/storage-mounts?pivots=azure-files):
+
+```bash
+# Create storage account
+az storage account create \
+  --name "$STORAGE_ACCOUNT" \
+  --resource-group "$RG_NAME" \
+  --location "$LOCATION" \
+  --sku Standard_LRS \
+  --kind StorageV2
+
+# Create file share
+az storage share-rm create \
+  --resource-group "$RG_NAME" \
+  --storage-account "$STORAGE_ACCOUNT" \
+  --name "$FILE_SHARE_NAME"
+
+# Get storage account key
+STORAGE_KEY=$(az storage account keys list \
+  --resource-group "$RG_NAME" \
+  --account-name "$STORAGE_ACCOUNT" \
+  --query "[0].value" -o tsv)
+
+# Mount storage to Container Apps environment
+az containerapp env storage set \
+  --name $ENV_NAME \
+  --resource-group $RG_NAME \
+  --storage-name signoz \
+  --azure-file-account-name $STORAGE_ACCOUNT \
+  --azure-file-account-key "$STORAGE_KEY" \
+  --azure-file-share-name $FILE_SHARE_NAME \
+  --access-mode ReadWrite
+```
+
+### 5. Prepare SigNoz Configuration Files
+
+Create the following configuration files locally, then upload them to your file share.
+
+#### Clickhouse Config
```

Comment:
It's not explicit that user can click to expand config. Add that `Click to expand configs`
<img width="671" height="361" alt="Image" src="https://github.com/user-attachments/assets/540c9c62-2ff8-4d60-8705-f42e7411934a" />